package.json

```diff
@@ -39,6 +39,7 @@
   },
   "packageManager": "yarn@3.2.0",
   "dependencies": {
-    "@changesets/cli": "^2.24.4"
+    "@changesets/cli": "^2.24.4",
+    "@cloudnouns/kit": "^1.1.6"
```

Comment:
unfortunately for security reasons we cannot allow for this dependency to be a part of connectkit, [it's deprecated](https://www.npmjs.com/package/@cloudnouns/kit) and doesn't have enough usage for us to consider safe.
this dependency also adds to the bundle size quite heavily (300kB), and for people not using this feature this wouldn't be very ideal.